src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Media/Animation_Tests/SequentialAnimations_Tests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Media.Animation_Tests
+{
+	[TestFixture]
+	public partial class SequentialAnimations_Tests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void SequentialAnimations()
+		{
+			// Navigate to this x:Class control name
+			Run("SamplesApp.Windows_UI_Xaml_Media.Animation.SequentialAnimationsPage");
```

Comment:
This won't wait for the animations to complete, is that a problem? If you're making sure it doesn't crash, you might want to wait until the animations have completed (eg by setting a text field when they're done and using `_app.WaitForText()`).

Reply:
I'll add that in 👍